action.yml

```
@@ -8,33 +8,37 @@ inputs:
   api-key:
     required: true
     description: Your Airplane API Key.
-  team-id:
-    required: true
-    description: Your Airplane Team ID.
+  env:
+    required: false
+    description: Airplane environment to deploy in.
+    default: ""
   host:
     required: false
     description: Airplane API Host.
     default: api.airplane.dev
-  env:
+  local:
     required: false
-    description: Airplane environment to deploy in.
+    description: If `"true"`, executes builds in the CI runner instead of in Airplane's infrastructure.
     default: ""
```

just to clarify the default behavior:

```suggestion
    default: "false"
```

---

Fixed.